codemon/CodemonHelp.py

```diff
@@ -1,11 +1,13 @@
 from clint.textui import colored
 
 def showHelp():

```

Comment:
Can you add two newlines right after the function definition ?

Reply:
Right after function definition?? Why is that so?

Reply:
@alphaX86 - basically I want two newlines above the logo and one newline after the logo